src/map.cpp

```diff
@@ -602,59 +602,49 @@ float map::vehicle_vehicle_collision( vehicle &veh, vehicle &veh2,
     // Vertical collisions will be simpler for a while (1D)
     if( !vertical ) {
         // For reference, a cargo truck weighs ~25300, a bicycle 690,
-        //  and 38mph is 3800 'velocity'
-        rl_vec2d velo_veh1 = veh.velo_vec();
-        rl_vec2d velo_veh2 = veh2.velo_vec();
+        //  Velocities are in 1/100 mph
+        // Masses are in kilograms
+        rl_vec2d velocity_veh1 = veh.velo_vec();
```

Comment:
while I am all for making variable names more descriptive, velo_veh1 was adequate and I feel this rename is unnecessary churn.